src/new-client/src/app/site/[site]/view-availability/week/page.tsx

```diff
@@ -0,0 +1,22 @@
+import NhsPage from '@components/nhs-page';
+import dayjs from 'dayjs';
+
+type PageProps = {
+  searchParams: {
+    from: string;
+    until: string;
+  };
+};
+
+const Page = async ({ searchParams }: PageProps) => {
+  return (
+    <NhsPage
+      title={`${dayjs(searchParams.from).format('D MMMM')} to ${dayjs(searchParams.until).format('D MMMM')}`}
+      breadcrumbs={[{ name: 'Home', href: '/' }]}
```

Comment:
I think we might be missing a breadcrumb here - shouldn't `/site/${params.site}/view-availability` be in here?

Reply:
I left this for now because it would be covered by APPt-338 as it was next on my list. Though admittedly I forgot to add a TODO to revisit it in that ticket. I'll add it in